packages/web-components/src/donut-chart/donut-chart.styles.ts

```diff
@@ -85,4 +91,41 @@ export const styles = css`
     line-height: ${lineHeightBase200};
     color: ${colorNeutralForeground1};
   }
+
+  .calloutContentRoot {
+    display: grid;
+    overflow: hidden;
+    padding: 11px 16px 10px 16px;
+    background-color: ${colorNeutralBackground1};
+    background-blend-mode: normal, luminosity;
+
+    border-radius: ${borderRadiusMedium};
+    border: 1px solid ${colorTransparentStroke};
+    filter: drop-shadow(0 0 2px ${colorNeutralShadowAmbient}) drop-shadow(0 8px 16px ${colorNeutralShadowKey});
+
+    position: absolute;
+    opacity: 0;
+    z-index: 1;
+    pointer-events: none;
+  }
+
+  .calloutBlockContainer {
+    padding-left: 8px;
+    color: ${colorNeutralForeground2};
+  }
+
+  .calloutLegendText {
+    font-family: ${fontFamilyBase};
```

Comment:
replace these group with typography defined in this file. https://github.com/microsoft/fluentui/blob/master/packages/web-components/src/styles/partials/typography.partials.ts

For eg all the font related styles here can be replaced with this.
https://github.com/microsoft/fluentui/blob/263ead68a05ac23499be137b41e02a5579522c08/packages/web-components/src/styles/partials/typography.partials.ts#L56

Reply:
done